.i18n/Game-zh_CN.po

```diff
@@ -0,0 +1,5296 @@
+msgid "For all natural numbers $m$, we have $ 0 \times m = 0$."
+msgstr "对于所有自然数 $m$，我们有 $ 0 \times m = 0$。"
+
+#: Game.Levels.Multiplication.L03succ_mul
+msgid "succ_mul"
+msgstr "succ_mul"
+
+#: Game.Levels.Multiplication.L03succ_mul
+msgid ""
+"\n"
+"Similarly we have `mul_succ`\n"
+"but we're going to need `succ_mul` (guess what it says -- maybe you\n"
+"are getting the hang of Lean's naming conventions).\n"
+"\n"
+"The last level from addition world might help you in this level.\n"
+"If you can't remember what it is, you can go back to the\n"
+"home screen by clicking the house icon and then taking a look.\n"
+"You won't lose any progress.\n"
+msgstr ""
+"\n"
+"同样，我们有 `mul_succ`，\n"
+"但我们需要 `succ_mul`（猜猜它是什么意思 —— 也许你已经掌握了 Lean 的命名范式）。\n"
+"\n"
+"加法世界中的最后一关会在这个关卡中帮助你。\n"
+"如果你忘记了它是什么，你可以通过点击房屋图标回到主界面，然后看一看。\n"
+"你不会失去任何进展。\n"
+
+#: Game.Levels.Multiplication.L03succ_mul
+msgid ""
+"`succ_mul a b` is the proof that `succ a * b = a * b + b`.\n"
+"\n"
+"It could be deduced from `mul_succ` and `mul_comm`, however this argument\n"
+"would be circular because the proof of `mul_comm` uses `mul_succ`."
+msgstr ""
+"`succ_mul a b` 是 `succ a * b = a * b + b` 的证明。\n"
+"\n"
+"可以从 `mul_succ` 和 `mul_comm` 推导出来，但是这个是循环论证，\n"
+"因为 `mul_comm` 的证明使用了 `mul_succ`。"
+
+#: Game.Levels.Multiplication.L03succ_mul
+msgid ""
+"For all natural numbers $a$ and $b$, we have\n"
+"$(\\operatorname{succ}\\ a) \times b = a\times b + b$."
+msgstr ""
+"对于所有自然数 $a$ 和 $b$，我们有\n"
+"$(\\operatorname{succ}\\ a) \times b = a\times b + b$。"
+
+#: Game.Levels.Multiplication.L04mul_comm
+msgid "mul_comm"
+msgstr "mul_comm"
+
+#: Game.Levels.Multiplication.L04mul_comm
+msgid ""
+"\n"
+"The first sub-boss of Multiplication World is `mul_comm x y : x * y = y * x`.\n"
+"\n"
+"When you've proved this theorem we will have \"spare\" proofs\n"
+"such as `zero_mul`, which is now easily deducible from `mul_zero`.\n"
+"But we'll keep hold of these proofs anyway, because it's convenient\n"
+"to have exactly the right tool for a job.\n"
+msgstr ""
+"\n"
+"乘法世界的第一个小 Boss 是 `mul_comm x y : x * y = y * x`。\n"
+"\n"
+"当你证明了这个定理后，我们将有一些“多余”的证明\n"
+"例如 `zero_mul`，它现在可以轻松地从 `mul_zero` 中推导出来。\n"
+"但无论如何我们都会保留这些证明，因为\n"
+"拥有适合工作的工具会很方便。\n"
+
+#: Game.Levels.Multiplication.L04mul_comm
+msgid ""
+"`mul_comm` is the proof that multiplication is commutative. More precisely,\n"
+"`mul_comm a b` is the proof that `a * b = b * a`."
+msgstr ""
+"`mul_comm` 是乘法可交换的证明。更确切地说，\n"
+"`mul_comm a b` 是 `a * b = b * a` 的证明。"
+
+#: Game.Levels.Multiplication.L04mul_comm
+msgid "Multiplication is commutative."
+msgstr "乘法是可交换的。"
+
+#: Game.Levels.Multiplication.L05one_mul
+msgid "one_mul"
+msgstr "one_mul"
+
+#: Game.Levels.Multiplication.L05one_mul
+msgid ""
+"\n"
+"You can prove $1\times m=m$ in at least three ways.\n"
+"Either by induction, or by using `succ_mul`, or\n"
+"by using commutativity. Which do you think is quickest?\n"
+msgstr ""
+"\n"
+"您可以至少通过三种方式证明 $1\times m=m$。\n"
+"通过感应（归纳法），或使用 `succ_mul`，或\n"
```

Comment:
```suggestion
"通过归纳法，或使用 `succ_mul`，或\n"
```

Reply:
I suppose I should ensure consistency throughout.